eZ/Publish/Core/MVC/Symfony/View/Builder/ContentViewBuilder.php

```diff
@@ -124,7 +124,9 @@ public function buildView(array $parameters)
                 throw new InvalidArgumentException('Content', 'Could not load any content from the parameters');
             }
 
-            $languageCode = $parameters['languageCode'] ?? null;
+            $mainRequest = $this->requestStack->getMainRequest();
+            $requestLanguageCode = $mainRequest === null ? null : $mainRequest->attributes->get('languageCode');
+            $languageCode = $parameters['languageCode'] ?? $requestLanguageCode;
```

Comment:
If parameters has `languageCode`, then there is no reason to get it from the `Request::attributes`. Move it into a method and change the order of operations.